### DISCUSSION TITLE
Update to 5.03 automatic sync option still grayed out

### DISCUSSION THREAD

**Discussion by visiLi** [floccusaddon/floccus#1477] [Q&A]
Google, browser
1.fix(options): Auto-sync option was not saved [Doesn't seem to be working]
2.Synchronization time is not a 15-minute interval, but always.[Synchronization in progress]
![image](https://github.com/floccusaddon/floccus/assets/42717403/e942cb35-49e9-4135-8b3a-6f9399ca6199)
![微信截图_20231214111925](https://github.com/floccusaddon/floccus/assets/42717403/339dfcf7-a228-4fdf-b2f4-2dbec5364694)

**Comment by marcelklehr** [comment]
Hello,

this is a discussion. If you encounter a problem in floccus that you would like the developers to solve, you should file a proper bug report as an *issue* not as a discussion and make sure to include all necessary information.
